app/src/main/java/de/westnordost/streetcomplete/quests/segregated/AddSegregated.java

```diff
@@ -0,0 +1,67 @@
+package de.westnordost.streetcomplete.quests.segregated;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.meta.OsmTaggings;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.powerpoles_material.AddPowerPolesMaterialForm;
+
+public class AddSegregated extends SimpleOverpassQuestType {
+
+	@Inject
+	public AddSegregated(OverpassMapDataDao overpassServer) {
+		super(overpassServer);
+	}
+
+	@Override
+	protected String getTagFilters() {
+		return
+			"ways with " +
+				"(" +
+				"(highway = path and bicycle = designated and foot = designated)" +
+				" or (highway = footway and bicycle = designated)" +
+				" or (highway = cycleway and foot = designated)" +
+				")" +
+				" and surface !~" + TextUtils.join("|", OsmTaggings.ANYTHING_UNPAVED) +
```

Comment:
or rather `surface ~ ...ANYTHING_PAVED`?

Reply:
This quest is after surface quest so either 

- surface is added (so unpaved should exclude, at least once #746 is fixed)
- user disabled surface quest

So the difference exists when

- user added unpaved, until #746 is fixed (not showing is preferable)
- surface is unpaved, user disabled surface quest (not showing is preferable)
- surface is paved, user disabled surface quest (showing is preferable)

So in the end I planned to convince you that `surface !~ ANYTHING_UNPAVED` is preferable but rather convinced myself.

Reply:
I will take list of paved surfaces from https://github.com/gravitystorm/openstreetmap-carto/blob/ace127056386dc7aa0dd0b9e428d049c986e5df7/project.mml#L500